src-test/org/etools/j1939tools/modules/GhgTrackingModuleTest.java

```diff
@@ -134,12 +133,12 @@ public void testFormat() {
         expected += "| Eng.Out.Energy, kW-hr   |      14,170 |      54,906 |   1,049,476 |" + NL;
         expected += "| PKE Numerator           |     180,735 | 811,401,221 |  51,163,080 |" + NL;
         expected += "| Urban Speed Run Time, s |       1,688 |           5 |   1,966,080 |" + NL;
-        expected += "| Idle Run Time, s        |         112 |       5,041 |         N/A |" + NL;
```

Comment:
I’m not sure about this change. If the packet is not supplied, the values should be N/A

Reply:
Entered new issue #114 to fix this.